docs/docsite/rst/os_guide/index.rst

```diff
@@ -18,10 +18,5 @@ Find out everything you need to know about using Ansible on Windows and with BSD
 .. toctree::
    :maxdepth: 2
 
-   windows_setup
-   windows_usage
-   windows_winrm
-   windows_dsc
-   windows_performance
-   windows_faq
-   intro_bsd
\ No newline at end of file
+   intro_bsd
+   intro_windows
```

Comment:
To resolve the [ci failures](https://github.com/ansible/ansible-documentation/actions/runs/10627072481/job/29459622206?pr=1820#step:5:57) I think you need to add references to files here.

You probably know this but you could add `:orphan:` to RST files you don't want to delete but don't want to appear in the index. They'll still get built by sphinx though.

Reply:
I might do that for some of the deleted files so we can at least redirect them. Do we need to add orphan files to a toctree to satisfy CI?